src/plugins/data/public/index.ts

```diff
@@ -42,6 +42,14 @@ export {
   // timefilter
   RefreshInterval,
   TimeRange,
+  // Field Formats
+  fieldFormats,
+  IFieldFormat,
+  FieldFormatsRegistry,
+  FieldFormatsContentType,
+  FieldFormatsGetConfigFn,
+  FieldFormatConfig,
+  FieldFormatId,
 } from '../common';
 export { autocomplete } from './autocomplete';
 export * from './field_formats';
```

Comment:
I don't think this should be here -- this is only exporting the field formats service, setup, and start types, none of which need to be public AFAICT